includes/course-theme/class-sensei-course-theme-template-selection.php

```diff
@@ -155,6 +156,30 @@ public static function get_templates(): array {
 
 		];
 
+		// Register a new learning mode version
+		if (Sensei()->feature_flags->is_enabled('learning_mode_v2')) {
```

Comment:
Instead of introducing it here we could use [this hook](https://github.com/Automattic/sensei/blob/9ed427ece0875e9a2ca341abd210c0e2a8e5654c/includes/course-theme/class-sensei-course-theme-template-selection.php#L195) to add the new template. 

It enables use to move the new learning mode logic to a specific module/class, instead of having it here, if we want.